js/main.js

```diff
@@ -201,10 +230,39 @@ function handleSubmit(event) {
       }
     }
   }
-
   $lyricsInput.value = '';
-  data.lyricCard++;
-  $arrowUp.className = 'pos-abs fas fa-angle-up';
+  $overlay.className = 'overlay';
+  if (data.lyricCard === data.totalLyricCards - 1) {
+    $finalTotalScore.textContent = 'Your final score: ' + data.runningScore + '/' + data.totalLyricCards * 2;
+    $modalSong.innerHTML = song + '<br>' + artist;
```

Comment:
**Yuck**. `innerHTML` is a dangerous property to use. Instead, you should generate a `br` element, then append `song`, that `br`, and `artist` to `$modalSong`. You can use `append` to add text content if it is in the same space as full elements, such as `br` or `span`